server/src/main/java/com/project/yozmcafe/domain/member/Member.java

```diff
@@ -49,15 +52,36 @@ public void removeUnViewedCafe(final UnViewedCafe unViewedCafe) {
         final UnViewedCafe foundUnviewedCafe = unViewedCafes.stream()
                 .filter(unViewedCafe::equals)
                 .findAny()
-                .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 내역입니다."));
-
+                .orElseThrow(() -> new IllegalArgumentException(UN_EXISTED_UNVIEWED_CAFE));
         unViewedCafes.remove(foundUnviewedCafe);
     }
 
     public boolean isEmptyUnViewedCafe() {
         return unViewedCafes.isEmpty();
     }
 
+    public void updateIsLike(final Cafe cafe, final boolean isLiked) {
+        final boolean currentIsLike = isLike(cafe);
+        if (currentIsLike != isLiked && isLiked) {
+            addLikedCafe(cafe);
+            cafe.addLikeCount();
+        }
+
+        if (currentIsLike != isLiked && !isLiked) {
+            removeLikedCafe(cafe);
+            cafe.subtractLikeCount();
+        }
+    }
+
```

Comment:
```suggestion
public void updateLikedCafesBy(final Cafe cafe, final boolean isLiked){
    if(alreadySatisfiedBy(cafe, isLiked)){
        return;
    }
    
    if(isLiked){
        addLikedCafe(cafe); //cafe.like();도 addLikedCafe(cafe) 내부에서 실행
    }
    
    removeLikedCafe(cafe); //cafe.unlike()도 removeLikedCafe() 내부에서 실행
}

private boolean alreadySatisfiedBy(final Cafe cafe, final boolean isLiked){
    final boolean isLikedCafe = likedCafes.stream()
                                                      .anyMatch(likedCafe -> likedCafe.isSameCafe(cafe));
    
    return isLikedCafe == isLiked;
}
```
1. 처음 봤을 때 `updateIsLike` 라는 네이밍이 다소 와닿지 않았던 것 같아요! 그래서 위와 같은 네이밍은 어떨까 생각했습니다.
2. 오션 말처럼 현재 상태가 이미 만족하는 지 보고 리턴하면 좋을 것 같습니다.
3. 카페의 카운트를 줄이는 것은 remove~, add~ 메서드에 포함되면 좋을 것 같아요. 생명주기가 같다고 생각합니다.

Reply:
1. 훨씬 좋은네이밍인 듯 해요!
2. 중론이 그러니 반영하겠습니다 ㅎㅎ
3. 오... 생각도 못했는데 백프로 동의합니다!